docs/configuration/configuration.md

```diff
@@ -2714,6 +2719,28 @@ anchored on both ends. To un-anchor the regex, use `.*<regex>.*`.
 Care must be taken with `labeldrop` and `labelkeep` to ensure that metrics are
 still uniquely labeled once the labels are removed.
 
+### `<scrape_rule_configs>`
+
+Scrape rules are applied after target relabeling and before metric relabeling. They can be used to evaluate PromQL expressions against samples from individual scrapes.
+
+Rules are only evaluated in the context of single scrapes and do not take into account samples from different targets, nor samples that are already ingested in Prometheus.
+
+Metric relabelings can be applied to series generated by scrape rules.
+
+A potential use of scrape rules is to aggregate away expensive series before they are ingested in Prometheus.
```

Comment:
the source series would need to be removed in this case. Is it worth adding this note here?

Reply:
Good idea, added a note.